src/components/explorer/BiosamplesTable.js

```diff
@@ -0,0 +1,165 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Link } from "react-router-dom";
+import { withBasePath } from "../../utils/url";
+import ExplorerSearchResultsTableComp from "./ExplorerSearchResultsTableComp";
+
+const biosampleRender = (individual, record) => {
+    const alternateIds = individual.alternate_ids ?? [];
+    const listRender = alternateIds.length ? " (" + alternateIds.join(", ") + ")" : "";
```

Comment:
Tough to read with all the quotes, a [string template](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) would be a better option here. There is also a white space before the parenthesis in " ("

Reply:
the parenthesis is ok, changes to string template done!